vignettes/useDataPkgs.Rmd

```diff
@@ -0,0 +1,219 @@
+package name is in lower case letters and the mapping names are in
+capital letters. When a given mapping maps platform specific keys to
+annotation data, only the name of the annotation data is used for the
+name of the mapping. Otherwise, the mapping names have a pattern of key
+name and value name joined by a \"2\" in between. For example,
+`hgu95av2ENTREZID` maps probe ids on an Affymetrix human genome U95Av2
+chip to EntrezGene IDs while `hgu95av2GO2PROBE` maps Gene Ontology IDs
+to probe IDs. Names of the mappings available in a platform specific
+data package are not listed here to save space but are easily accessible
+as shown later in the section for usage.
+
+Genome level annotation libraries are named in the form of org.Xx.yy.db
+where Xx represents an abbreviation of the genus and species. Each of
+the organism wide genome anotation packages is based upon some type of
+widely used gene based identifier (such as an Entrez Gene id) that is
+mapped onto all the other features in the package. The yy part of the
+name corresponds to this designation, where eg means a package is an
+entrez gene package and sgd is a package based on the sgd database etc.
+In many cases the org packages will contain more different kinds of
+information that the platform based ones, since not all types of
+information are as widely sought after.
+
+The *KEGG.db* library contains mappings between ids such as Entrez Gene
+IDs and *GO* to *KEGG* pathway ids and thus also to pathway names. The
+*GO.db* library maintain the directed acyclic graph structure of the
+original data from Gene Ontology Consortium by providing mappings of GO
+ids to their direct parents or children for each of the three categories
+(molecular function, cellular component, and biological process).
+Mappings between Entrez Gene and *GO* ids are also available to
+complement the the GO.db package. These mappings are found within the
+organism wide packages mentioned above. These mappings are provided with
+evidence code that specifies the type of evidence that supports the
+annotation of a gene to a particular *GO* term.
+
+# Usage
+
+All the annotation libraries can be obtained from the Bioconductor web
+site ((http://www.bioconductor.org)). To illustrate their usages, we use
+the library for Affymetrix HG_U95Av2 chip (*hgu95av2.db*) as an example
+for platform specific data packages and the *GO.db* library for
+non-platform specific data packages. We assume that R
+((www.r-project.org)) and Bioconductor's Biobase and annotation
+libraries have already been installed.
+
+## Package installation {#package-installation .unnumbered}
+
+Download libraries *hgu95av2.db* and *GO.db* with
+BiocManager::install(). If the *reposTools* library has already been
+installed/loaded, typing `install.packages2(library name)` installs the
+library for both Unix and Windows.
+
+Typing library(library name) in an R session will load the library into
+R. For example,
+
+```{r loadLibs>>=
```

Comment:
These might be causing some problems. Code blocks should look as follows

    ```{r nameOfCodeBlock}
    # some code should be written here for example
    a <- 1 + 2
    ```
After the three backticks, we have an opening `{` followed by `r`, which describes the type of code (`R` language). Then we have the name then usually some settings for the code bloc that tell the `knitr` package how to process the code block. For example, should it be run or hidden from the user? Then we should have a closing`}` before the content of the code block.

In the `.Rnw` Sweave file, code blocks start with `<nameOfCodeBlock>>=`, which you can see by looking at the corresponding code block in the .Rnw at https://github.com/Bioconductor/annotate/pull/6/files#diff-054eecc2290790ab7c39d71de080e62d1bc357e3769176f3401a14819ba80c61L128-L132.

You should replace the `>>=` with `}`. There are other places in this document that need replacement.